react/src/base/inputs/SprkTextInput/SprkTextInput.test.js

```diff
@@ -146,4 +153,57 @@ describe('SprkTextInput:', () => {
     const wrapper = mount(<SprkTextInput helperText="" />);
     expect(wrapper.find('.sprk-b-HelperText').length).toBe(0);
   });
+
+  it(`should have aria-describedby if the field is invalid and
```

Comment:
Is this code still what we want or did you discover any changes needed as part of react input work that are yet to be completed @guern1kn

Reply:
This is better than nothing, but this is going to be changed with the React Input API work.

Reply:
@guern1kn  as long as it accomplishes what we want and doesn't cause errors or issues. Is that true?

Reply:
Correct. This was the first iteration of it. It at least adds `aria-describedby` if there's an error container. It will eventually be updated to support helper text as well.